File: solutions/4074-count-subarrays-with-majority-element-i/solution.py
class Solution:
    def countMajoritySubarrays(self, nums: List[int], target: int) -> int:
        n = len(nums)
        count = 0
        for i in range(n) :
            appears = 0
            for j in range(i, n) :
                if (nums[j] == target) :
                    appears += 1
                subarray = j - i + 1
                if (subarray / 2 < appears) :
                    count += 1
        return count

File: solutions/4074-count-subarrays-with-majority-element-i/solution.java
class Solution {
    public int countMajoritySubarrays(int[] nums, int target) {
        int count = 0;
        for(int i = 0; i < nums.length; i++) {
            int appears = 0;
            for(int j = i; j < nums.length; j++) {
                if (nums[j] == target) {
                    appears += 1;
                }
                if ((j - i + 1) / 2 < appears) {
                    count += 1;
                }
            }
        }
        return count;
    }
}
